### DISCUSSION TITLE
Typescript problem: onClickOutside: event.target in callback

### DISCUSSION THREAD

**Discussion by rudimadima** [vueuse/vueuse#1959] [Q&A]
Hi!
My code:
```typescript
const dropdownElem = ref();
const menuButtonElem = ref();
onClickOutside(dropdownElem, (event) => {
  if (menuButtonElem.value.contains(event.target)) return; // TypeScript problem here
  isDropdownOpen.value = false;
});
```
Typescript highlights with red color .target  with error message:
```
Property 'target' does not exist on type 'E'
```
But property `target` exists in event and code works fine.
I don't want to write `(event: any)`.
How can I solve this problem?

**Comment by sibbng** [comment]
My bad, I just created a PR to fix it: https://github.com/vueuse/vueuse/pull/1961
